main/solution/backend/config/infra/functions.yml

```diff
@@ -42,6 +42,7 @@ envStatusPollHandler:
     APP_CUSTOM_USER_AGENT: ${self:custom.settings.customUserAgent}
     APP_IS_APP_STREAM_ENABLED: ${self:custom.settings.isAppStreamEnabled}
     APP_DEFAULT_IDP_TYPE: ${self:custom.settings.defaultIdpType}
+    APP_AWS_SUFFIX: ${self:custom.settings.awsSuffix}
```

Comment:
Anywhere is using this variable?

Reply:
using this variable for assuming role through STS
https://github.com/awslabs/service-workbench-on-aws-cn/blob/develop/addons/addon-base/packages/services/lib/aws/aws-service.js#L95